ci/prow/check-commit-content

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+pr_title=$(gh pr view ${PULL_NUMBER} -R ${REPO_OWNER}/${REPO_NAME} --json title --jq '.title')
+# Github UI adds '^M' (which is interpreted as '\r' by the 'sed' command) at the end of lines.
+pr_body=$(gh pr view ${PULL_NUMBER} -R ${REPO_OWNER}/${REPO_NAME} --json body --jq '.body' | sed 's/\r$//g')
+# We assume that there is a single commit per PR. It is validated in another job.
```

Comment:
Should we merge both jobs?

Reply:
Let's first make sure that https://github.com/kubernetes-sigs/kernel-module-management/pull/34 passes CI to validate that the job is able to consume the Github token from Prow. Next we move to this job (we will need to merge https://github.com/kubernetes-sigs/kernel-module-management/pull/34 for it first).